### PR TITLE
[dpp] update from 10.0.15 to 10.0.16

### PR DESCRIPTION
**Pull request by braindigitalis**
- #### What does your PR fix?
Upgrades dpp package from 10.0.15 to 10.0.16.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes

### CODE REVIEW

File: versions/d-/dpp.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "versions": [
     {
-      "git-tree": "fe4fafd238e318a6ddf7d5c4db59da499328fdae",
-      "version": "10.0.15",
+      "git-tree": "55f6414fcc02c066f5c224274cbf5b5b5ab0abc3",
+      "version": "10.0.16",
```
Comment (Thomas1664) [inline versions/d-/dpp.json:5]: Please don't modify an existing version entry in this file. Revert this and use `vcpkg x-add-version dpp` instead.

File: ports/dpp/portfile.cmake [hunk 2]
```diff
@@ -29,4 +31,4 @@ file(
     RENAME copyright
 )
 
-file(COPY "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
```
Comment (Thomas1664) [inline ports/dpp/portfile.cmake:34]: Please add a final newline.

File: ports/dpp/portfile.cmake [hunk 3]
```diff
@@ -1,8 +1,10 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO brainboxdotcc/DPP
-    REF d9eb0083c7d4f3bb188ac4fde12f3af9a4443696
-    SHA512 5b6514a1152d566bba8959f2e51284142d57eec086e634a2b33a08186ee5cb5800a6244d92b9c8e367b60a542ed75b4401f4d68b510f16226633b958aa4aeefa
+    REF  
+f94335e48a608e7eb8023e0a8268c33cefcd9737
+    SHA512  
+febe26fbf092331c1b877ed3e7e10d8176b089e5bded71144b96f17403bd4cc9990b0940873fadc16b5b6c2dd110c6ad2a1fb66534b73a40939d80d9c36d2a17
```
Comment (JonLiu1993) [inline ports/dpp/portfile.cmake:7]: @braindigitalis, thanks for your PR, please keep the indentation the same as the previous version

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/dpp/portfile.cmake [hunk 4]
```diff
@@ -1,8 +1,10 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO brainboxdotcc/DPP
-    REF d9eb0083c7d4f3bb188ac4fde12f3af9a4443696
-    SHA512 5b6514a1152d566bba8959f2e51284142d57eec086e634a2b33a08186ee5cb5800a6244d92b9c8e367b60a542ed75b4401f4d68b510f16226633b958aa4aeefa
+    REF  
+8fbeee90d1c0977026a319ccbdcc619d6580925a
+    SHA512  
+20aa3846f3b4c68a78b57ea57f4fe71e2257dc978c5c16448763ce605d4c29af20dd5780bb40c06afb043508fe4ea650e56ff52dbc021e1e47d31573791a5bea
```
Comment (JonLiu1993) [inline ports/dpp/portfile.cmake:7]:
```suggestion
    REF  8fbeee90d1c0977026a319ccbdcc619d6580925a
    SHA512  20aa3846f3b4c68a78b57ea57f4fe71e2257dc978c5c16448763ce605d4c29af20dd5780bb40c06afb043508fe4ea650e56ff52dbc021e1e47d31573791a5bea
```

File: ports/dpp/portfile.cmake (hunk 3 above)
Comment (braindigitalis) [reply ports/dpp/portfile.cmake]: these are auto generated. i will take a look but not sure what i can do.

Comment (braindigitalis) [reply ports/dpp/portfile.cmake]: fix for this should push in the next few minutes. thanks!

Comment (JonLiu1993) [reply ports/dpp/portfile.cmake]: I will try on my machine

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**